app/scripts/modules/core/src/pipeline/config/triggers/triggers.html

```diff
@@ -18,6 +18,21 @@
       </div>
     </div>
   </page-section>
+  <page-section key="artifacts" label="Artifacts" badge="pipeline.expectedArtifacts.length" no-wrapper="true">
```

Comment:
I'd call this section `Expected Artifacts`.

Reply:
I'd also consider moving this down on the page (probably below parameters or notifications), especially since we don't have much support for artifacts yet. I'd also feature flag this.